pillar/elife-public.sls

```diff
@@ -38,11 +42,43 @@ elife:
                 - chris
 
             # per-user, per-project access
+            containers:
+                - alfred
+                - daniel
+                - daniel_inspiron
+                - daniel_xps
+                - nlisgo
+            crm:
+                - graham_dev
+                - graham_ubuntu_vm
+                - alfred
+                - michael
+                - nlisgo # turning on instances for journal-cms testing
+                # temporary, Daniel needs to run once-off SQL
+                - daniel
+                - daniel_inspiron
+                - daniel_xps
+                - circle_interactive
             elife-xpub:
                 - alfred
                 - tamlyn
                 - hem
                 - peterhooper
+            peerscout:
+                - daniel
+                - daniel_inspiron
+                - daniel_xps
+                - alfred
+            sciencebeam-texture:
+                - daniel
+                - daniel_inspiron
+                - daniel_xps
+                - alfred
+            data-pipeline:
```

Comment:
alfred doesn't need to be added to the `data-pipeline`?

Reply:
it wasn't present before, we'll add it when there is a pipeline that needs it (hint: we should create a pipeline)